src/pipelines/master_pipeline.py

```diff
@@ -171,17 +177,17 @@ def get_args(
         if LDA_COS: lda['cos'] = LDA_COS
         if LDA_VOCAB: lda['vocab'] = LDA_VOCAB
         if LDA_TOPICS: lda['topics'] = LDA_TOPICS
-    
```

Comment:
Unless wordonehot will be handled differently than the other features, suggest you consider adding onehotword into the logic at lines 157-188 in order to pass in its parameters via the features dictionary. Otherwise I think onehotword won't get triggered in data_gen.py because 'if 'wordonehot' in features' will return False.